src/discussions/posts/data/slices.js

```diff
@@ -97,7 +76,11 @@ const threadsSlice = createSlice({
     },
     postThreadSuccess: (state, { payload }) => {
       state.postStatus = RequestStatus.SUCCESSFUL;
-      normaliseThreads(state, [payload]);
+      state.threadsById[payload.id] = payload;
+      state.threadsInTopic[payload.topicId].push(payload.id);
+      // Temporarily add it to the top of the list so it's visible
+      state.pages[1] = [payload.id] + (state.pages[1] || []);
```

Comment:
this is now not making sense with the above logic.

Reply:
Fixed.

Reply:
Thanks. it resolves my concern